photon-ml/src/main/scala/com/linkedin/photon/ml/model/GAMEModel.scala

```diff
@@ -112,3 +147,17 @@ class GAMEModel(gameModels: Map[String, DatumScoringModel]) extends DatumScoring
     super.hashCode()
   }
 }
+
+object GAMEModel {
+
+  /**
+   * Factory method to make code more readable.
+   *
+   * @param taskType The model type for this GameModel
+   * @param sections The sections that make up this GameModel
+   * @return A new instance of GAMEModel
+   */
+  def apply(taskType: TaskType, sections: (String, DatumScoringModel)*): GAMEModel = {
+    new GAMEModel(Map(sections:_*), taskType)
+  }
+}
```

Comment:
If you look in the diff section of GitHub you'll see a little icon on line 163 that represents a missing newline char - if you check your code, you'll see that it ends on line 163, but there should be an empty line 164